### PR TITLE
Feature/discord message chunking

### PR DESCRIPTION
**Pull request by Awarua-**

### CODE REVIEW

File: core/core_integrations/discord.js [hunk 1]
```diff
@@ -80,14 +107,25 @@ let discord = require('discord.js'),
...
+        var message = messageList[0];
+        messageList.splice(0, 1);
+        sendSingleMessage(message, threadId, channel, sendMultipleMessages.bind(this, messageList, threadId, channel));
+    },
+
     sendMessage = function(message, threadId) {
         let channel = lookUpChannel(threadId);
         message = addMentions(message);
-        if (message.length > 2000) {
-            message = $$`Sorry, output was too long to send :(`;
```
Comment (mrkno) [inline core/core_integrations/discord.js]: Remove translation that is no longer used

Comment (Awarua-) [inline core/core_integrations/discord.js]: Done

File: core/core_integrations/discord.js [hunk 2]
```diff
@@ -80,14 +108,25 @@ let discord = require('discord.js'),
...
+    sendSingleMessage = function(message, threadId, channel, callback) {
+        stopTyping(threadId);
+        bot.sendMessage(channel, message, callback);
+    },
+
+    sendMultipleMessages = function(messageList, threadId, channel) {
+        if (!messageList || messageList.length <= 0) {
+            return;
+        }
+        var message = messageList[0];
```
Comment (mrkno) [inline core/core_integrations/discord.js]: let if you are being consistent

**Comment by ParasDPain** [comment]
Very neat and readable code 👍 👍 Great job!

**Comment by mrkno** [comment]
Approved

**Pull request merged**